homeassistant/components/denonavr/manifest.json

```diff
@@ -1,7 +1,22 @@
 {
   "domain": "denonavr",
   "name": "Denon AVR Network Receivers",
+  "config_flow": true,
   "documentation": "https://www.home-assistant.io/integrations/denonavr",
-  "requirements": ["denonavr==0.8.1"],
-  "codeowners": ["@scarface-4711", "@starkillerOG"]
+  "requirements": ["denonavr==0.8.1", "getmac==0.8.2"],
```

Comment:
could be changed to `denonavr==0.9.0` now
same for requirements_all.txt and requirements_test_all.txt

Reply:
done